### PR TITLE
Create Write() in TestContext

### PR DESCRIPTION
**Pull request by HeroMaxPower**
Created Write() method.
Created some tests for the testing of newline and carriage return.

It's basically a carbon copy of the WriteLine implementation but calling the StringWriter.Write 

Resolves Issue:
#645

NOTE: I'm pretty inexperienced in C# i might struggle with advanced concepts if i've missed something.

### CODE REVIEW

**Comment by HeroMaxPower** [comment]
Sorry i missed some implementations, this should be ready for review now.

File: test/UnitTests/PlatformServices.Desktop.Unit.Tests/Services/DesktopTestContextImplTests.cs [hunk 1]
```diff
@@ -215,6 +215,76 @@ public void AddResultFileShouldAddMultipleFilestoResultsFiles()
...
+        }
+
+        [TestMethod]
+        public void WriteWithMessageShouldWriteToStringWriterForReturnCharacters()
+        {
+            var stringWriter = new StringWriter();
+            this.testContextImplementation = new TestContextImplementation(this.testMethod.Object, stringWriter, this.properties);
+            this.testContextImplementation.Write("2 Testing write \n\r");
+            this.testContextImplementation.Write("3 Testing write\n\r");
+            StringAssert.Equals(stringWriter.ToString(), "2 Testing write 3 Testing write");
```
Comment (nohwnd) [inline test/UnitTests/PlatformServices.Desktop.Unit.Tests/Services/DesktopTestContextImplTests.cs:285]: why are the new line characters not present in the output? I thought the difference between `Write` and `WriteLine` are that the former adds newline character at the end of the provided message, while `Write` does not.

Comment (HeroMaxPower) [reply test/UnitTests/PlatformServices.Desktop.Unit.Tests/Services/DesktopTestContextImplTests.cs:285]: Best example is [here](https://www.tutorialspoint.com/What-is-the-difference-between-Write-and-WriteLine-methods-in-Chash), the StringAssert checks that a Newline is not present in a string writer.

Comment (HeroMaxPower) [reply test/UnitTests/PlatformServices.Desktop.Unit.Tests/Services/DesktopTestContextImplTests.cs:285]: Also im calling the .Write() implementation in the StringWriter so really this is a wrapper similar to WriteLine() in StringWriter.

**Pull request merged**